testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py

```diff
@@ -551,6 +551,82 @@ def test_delta_sync_nested_doc(params_from_base_test_setup, num_of_docs, replica
     verify_delta_stats_counts(sg_client, sg_admin_url, replication_type, sg_db, num_of_docs)
 
 
+@pytest.mark.listener
+@pytest.mark.syncgateway
+@pytest.mark.replication
+@pytest.mark.parametrize("num_of_docs, replication_type", [
```

Comment:
not needed, as there is only one case involved. Unless we are planning to add more option in it.